### DISCUSSION TITLE
List of tokens / shortcodes / flags like for DD/MM/YY

### DISCUSSION THREAD

**Discussion by ewebgh33** [audioscavenger/save-image-extended-comfyui#32] [General]
Hi
Where can I find a list of tokens that this node will accept?
eg the default has this pre-filled:
%F %H-%M-%S
Where it looks like %F is date and HMS is hours minutes seconds.
Are there others?

**Comment by audioscavenger** [comment]
all is detailed here https://www.man7.org/linux/man-pages/man1/date.1.html

this is also indicated in the help button (the "?" on the top right of the node)
